CameraOverlay/tests/mqtt_message_test.py

```diff
@@ -0,0 +1,61 @@
+""" Test Script That Sends A Message To the Camera Overlay """
+import argparse
+import paho.mqtt.client as mqtt
+import time
+
+def get_args(argv=None):
+    """Get arguments passed into Python script"""
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--host", type=str, default="localhost", help="ip address of the host")
+    parser.add_argument("--waitTime", type=int, default=8, help="Number of seconds until next message is sent")
+    return parser.parse_args(argv)
+
+class MessageTest():
+
+    def __init__(self, host_ip, wait_Time):
+        self.host_ip = host_ip
+        self.wait_Time = wait_Time
+        self.mqtt_client = None
+        self.message_topic = '/v3/camera/primary/message'
```

Comment:
You already have all the topics defined in `CameraOverlay/topics.py`, you could use those

Reply:
Turns out we don't have access to `topics` here, as it is in a parent directory which cannot be accessed when this script is ran directly (https://chrisyeh96.github.io/2017/08/08/definitive-guide-python-imports.html). Anyone got ideas on how we could structure this better so that this isn't an issue? I haven't thought of any at this point. @angustrau @khanguslee

Reply:
On second thought as this is a test, it might be better to not use the assumed topics from the main program, which might be wrong.

Reply:
I encountered this problem in BOOST. See https://github.com/monash-human-power/BOOST/tree/master/tests where all I had to do was add an `__init__.py` file to the `tests` folder. This gave me access to the parent folder. I've forgotten how this works.

Reply:
@khanguslee  I already have an  __init__.py file in the tests folder

Reply:
I got no idea then, I think a better solution would be to have a `--primary` and `--secondary` argument so that we can set the topic to the desired camera overlay.